### DISCUSSION TITLE
Why there is no FLAC encoding presets?!

### DISCUSSION THREAD

**Discussion by deton24** [Tichau/FileConverter#132] [Q&A]
(Like above)

**Comment by Tichau** [comment]
Hello,
Conversion to FLAC is set only for the following file extensions: 
![image](https://user-images.githubusercontent.com/3792144/102011747-9747c400-3d46-11eb-93ee-1c3061b03f76.png)

This is the default configuration because I think it is not a good idea (in my opinion) to convert a compressed format to FLAC since it will not improve the audio quality and it will use more space on your drive.
If you want to do it anyway, you can edit the preset to add the input format you want in the "To Flac" preset.
